doc/man7/pbkdf2.pod

```diff
@@ -0,0 +1,129 @@
+=pod
```

Comment:
We had a discussion over where general algorithm descriptions should go. Should we hang them off a (possibly obscure) ctrl operation manual page or reference the algorithm name itself? IIRC the conclusion was the algorithm name was better and it should go in section 7.